File: Auto-generated-tasks/Task-12/Solution.f90
integer:: Count = 6, i1, Znach = 1
double precision(len=Count):: S
do i1 = 0, Count
    if (S(i1) > S(Znach)) then
        Znach = i1
    end if
end do
print *, "Znach=", Znach

File: Auto-generated-tasks/Task-12/Solution.pas
Var
	Count := 6, i1, Znach := 1: integer;
	S:Array[0..Count] of real;
Begin
	for i1:= 0 to Count do begin
		if (S[i1] > S[Znach]) then begin
			Znach := i1;
		end;
	end;
	WriteLn("Znach=", Znach);
End.